ResearchKit/ActiveTasks/ORKAccelerometerRecorder.h

```diff
@@ -49,12 +49,14 @@ ORK_CLASS_AVAILABLE
 /**
  Returns an initialized accelerometer recorder using the specified frequency.
  
+ @param identifier   The unique indentifier of the recorder (assigned by the recorder configuration).
  @param frequency    The frequency of accelerometer data collected from CoreMotion, in hertz (Hz).
  @return An initialized accelerometer recorder.
```

Comment:
Can you also document the other parameters?

@param step    The step that requested this recording.
@param outputDirectory    The directory in which the _______ should be stored.